src/lake/Lake/Build/Library.lean

```diff
@@ -73,6 +73,20 @@ def LeanLib.staticFacetConfig : LibraryFacetConfig staticFacet :=
 def LeanLib.staticExportFacetConfig : LibraryFacetConfig staticExportFacet :=
   mkFacetJobConfig (LeanLib.recBuildStatic · true)
 
+@[specialize] protected def LeanLib.buildStaticFat
+(self : LeanLib) : FetchM (BuildJob FilePath) := do
+    withRegisterJob s!"{self.name}:static.fat" do
+    let mods ← self.rootModules.concatMapM fun mod => do
+      return (← mod.transImports.fetch).push mod
+    let oJobs ← mods.concatMapM fun mod =>
+      mod.nativeFacets (shouldExport := true) |>.mapM fun facet => fetch <| mod.facet facet.name
```

Comment:
Should the fat library really have exported symbols? Exported symbols are generally only for dynamic libraries. Statically linking a library will cause symbol resolution regardless of export.